src/main/resources/tags/sensitive_materials.json

```diff
@@ -1,11 +1,33 @@
 {
 	"values" : [
 		"#minecraft:saplings",
-		"#minecraft:wooden_pressure_plates",
 		"#slimefun:torches",
-		"minecraft:stone_pressure_plate",
-		"minecraft:light_weighted_pressure_plate",
-		"minecraft:heavy_weighted_pressure_plate",
-		"minecraft:cake"
+		"#minecraft:buttons",
+		"#minecraft:signs",
+		"#slimefun:mushrooms",
+		"#minecraft:doors",
+		"minecraft:bamboo",
+		"#minecraft:rails",
+		"#minecraft:small_flowers",
+		"#minecraft:coral_blocks",
+		"#minecraft:coral_plants",
+		"#minecraft:small_flowers",
+		"#minecraft:banners",
+		"minecraft:sugar_cane",
+		"minecraft:ladder",
+		"minecraft:snow",
+		"minecraft:lever",
+		"minecraft:repeater",
+		"minecraft:tripwire_hook",
+		"minecraft:tall_grass",
+		"minecraft:large_fern",
+		"minecraft:lily_pad",
+		"minecraft:dead_bush",
+		"minecraft:fern",
+		"minecraft:grass",
+		"minecraft:seagrass",
+		"minecraft:tall_seagrass",
+		"minecraft:cactus",
+		"minecraft:kelp"
```

Comment:
Why the hell did you move these here :NotLikeThis: They should have stayed in the block placer tag...

Reply:
these all break when ever the block gets removed

Reply:
They are way too many and not even relevant for this.
This will lag out the server immensely.

Reply:
ah okay will remove it